.travis.yml

```diff
@@ -3,12 +3,17 @@ language: node_js
 node_js:
   - "16"
```

Comment:
why no `language: java`

Reply:
hey, there is already `language: node_js` at the top of the file